content/en/about/faq.md

```diff
@@ -146,7 +146,7 @@ We would love that. The more people monitoring the logs and providing useful ser
 
 There's no need for a distributed source of transparency as there can be multiple points of transparency which only adds more sources of security guarantee, not fewer. An entity can post to as many Rekor logs as they want and inform users of where they post. We do encourage folks to use common public instances, but we don't seek to enforce this. We do plan to look to produce a gossip protocol, for those that desire a more decentralised model (if there's demand).
 
-### How do I verify downloaded code?
+### Why not use a blockchain?
```

Comment:
Actually, what do you think about deleting this section entirely? It's a good discussion topic, but it's quite technical for an FAQ and I think the answer here is a bit too concise and maybe even a bit inaccurate.

Reply:
Sounds good to me, we can always re-add it later if necessary